build-tools/xaprepare/xaprepare/Application/Context.cs

```diff
@@ -329,6 +329,15 @@ partial class Context : AppObject
 		/// </summary>
 		public RefreshableComponent ComponentsToRefresh { get; set; }
 
+
+		public bool BuildingAZPPullRequest {
```

Comment:
This doesn't "look right" to my eyes; the "AZZP" is off-putting, and I'm not quite sure what it's supposed to mean.  I'd prefer `BuildingAzurePullRequest`.

Reply:
Seconded :)

Reply:
Agreed :)

Reply:
I've updated this to use the suggested name.